tests/test_acasclient/test_045_register_sdf_case_insensitive.sdf

```diff
@@ -1,28 +1,41 @@
-NSC 1390
-
-
- 10 11  0  0  0  0  0  0  0  0999 V2000
-   -4.4591   -4.9405    0.0000 N   0  0  0  0  0  0  0  0  0  0  0  0
-   -3.1600   -2.6905    0.0000 N   0  0  0  0  0  0  0  0  0  0  0  0
-   -3.1600   -7.1905    0.0000 O   0  0  0  0  0  0  0  0  0  0  0  0
-   -0.4344   -2.9770    0.0000 N   0  0  0  0  0  0  0  0  0  0  0  0
-    0.4473   -4.1905    0.0000 N   0  0  0  0  0  0  0  0  0  0  0  0
-   -1.8610   -3.4405    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
-   -1.8610   -4.9405    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
-   -3.1600   -5.6905    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
-   -0.4344   -5.4040    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
-   -4.4591   -3.4405    0.0000 C   0  0  0  0  0  0  0  0  0  0  0  0
+
```

Comment:
Modified the compounds in this file because I didn't want these structures interfering with the structures in the other test file.  i.e. load the one and then load the other one and they should get different parent ids.